### ISSUE TITLE
Topic classification

### ISSUE THREAD

**Issue by BrisaIs** [Sid200026/ToxicBot#3]

**Comment by Sid200026** [comment]
Can you elaborate ?

**Comment by Sid200026** [comment]
Can you elaborate ?

**Comment by BrisaIs** [comment]
> Can you elaborate ?

Yeah, sorry, I mean to classificate the message for knowing what kind of harassment is, like if its racial, lgtb related, sexual or so, what do you think ?

**Comment by Sid200026** [comment]
Ok. Sounds good to me. Go ahead
